photon-core/src/main/java/org/photonvision/vision/pipeline/ArucoPipeline.java

```diff
@@ -96,7 +97,8 @@ protected CVPipelineResult process(Frame frame, ArucoPipelineSettings settings)
         CVPipeResult<List<ArucoDetectionResult>> tagDetectionPipeResult;
 
         if (rawInputMat.empty()) {
-            return new CVPipelineResult(sumPipeNanosElapsed, 0, List.of(), frame);
+            return new CVPipelineResult(
+                    sumPipeNanosElapsed, 0, List.of(), new MultiTargetPNPResults(), frame);
         }
```

Comment:
Looking at the `CVPipelineResult` constructor changes, this class shouldn't need any modifications.

Reply:
How should MultiTargetPNPResults end up getting sent over NT if not through CVPipelineResult?

Reply:
What I mean is that these classes are modified to construct an empty `MultiTargetPNPResults`, but the `CVPipeline` constructor does that already when it is omitted (if it was just `new CVPipelineResult(sumPipeNanosElapsed, 0, List.of(), frame)`).